mapie/calibrators/ccp/utils.py

```diff
@@ -35,17 +35,7 @@ def format_functions(
         If ``None``, return an empty list.
 
     bias: bool
-        Add a column of ones to the features, for safety reason
-        (to garanty the marginal coverage, no matter how the other features
-        the ``CCPCalibrator``object were built).
-        If the ``CCPCalibrator``object definition covers all the dataset
-        (meaning, for all calibration and test samples, the resulting
-        ``calibrator.predict(X, y_pred, z)`` is never all zeros),
-        this column of ones is not necessary to obtain marginal coverage.
-        In this case, you can set this argument to ``False``.
-
-        If you are not sur, use ``bias=True`` to garantee the marginal
```

Comment:
Why do we no longer use a default value?

Reply:
In the calibrator initialisation, `bias=True` by default, but in this check function, bias is always define, so don't need a default value. Or did I not understood ?